### ISSUE TITLE
Interactive project creation: space to select does not work

### ISSUE THREAD

**Issue by willowmck** [halkyonio/hal#14]
> hal project my-new-project
? Spring Boot version [Use arrows to move, space to select, type to filter]

Pressing the space bar on the default selection removes the list.

**Comment by metacosm** [comment]
Indeed. This looks like a bug in the underlying library: despite what the message is saying, space is considered as a character to filter with (at least, that's what it looks like). Pressing enter works fine to select.
